### PR TITLE
Remove the User Model, give it an interface to allow easy customization

### PR DESCRIPTION
**Pull request by gitlabbin**

### CODE REVIEW

**Comment by gitlabbin** [comment]
Use this way to fix the issue #12 as well

**Comment by bassaer** [comment]
Thanks!
let me check

File: chatmessageview/src/main/kotlin/com/github/bassaer/chatmessageview/model/IChatUser.kt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.github.bassaer.chatmessageview.model
+
+import android.graphics.Bitmap
+
+interface IChatUser {
+    fun getId(): String
+    fun getName(): String?
+    fun getIcon(): Bitmap?
+    fun setIcon(bmp: Bitmap)
+}
```
Comment (bassaer) [inline chatmessageview/src/main/kotlin/com/github/bassaer/chatmessageview/model/IChatUser.kt]: please add newline here

Comment (gitlabbin) [reply chatmessageview/src/main/kotlin/com/github/bassaer/chatmessageview/model/IChatUser.kt]: Didn't run the test, will clean it up

**Comment by bassaer** [comment]
looks good
Thanks!

**Pull request merged**